docs/quickstart/README.md

```diff
@@ -199,7 +199,7 @@ spec:
             periodSeconds: 5
         ### 👈 BEGIN ADDITIONAL CONTAINER ###
         - name: tornjak-backend
-          image: ghcr.io/spiffe/tornjak-backend:latest
+          image: ghcr.io/spiffe/tornjak-backend:2.0.0
```

Comment:
Shouldn't we keep the image the the latest, instead of updating everytime? Or is this temo?

Reply:
I think we haven't been updating latest - only tagging with the git sha and the version
let's make an issue for this